### PR TITLE
Manifest v2

### PR DESCRIPTION
**Pull request by gabriel-okteto**

### CODE REVIEW

File: okteto.yml [hunk 1]
```diff
@@ -1,9 +1,41 @@
-name: hello-world
-image: okteto/hello-world:ruby-dev
-command: [ "bash" ]
-workdir: /opt/app/
-forward:
-  - 8080:8080
-  - 1234:1234
-persistentVolume:
-  enabled: false
+name: ruby-getting-started
```
Comment (pchico83) [inline okteto.yml]: Let's remove comments and unneeded fields, as we do for this sample:
https://github.com/okteto/go-getting-started/pull/10/files#diff-dd8070292ce806a326a3d0543b136ba3b5f7930f40d33475780462fdcece2b70

File: .gitignore [hunk 2]
```diff
@@ -1 +1,4 @@
 vendor/bundle
+
+.DS_Store
```
Comment (pchico83) [inline .gitignore:3]: You are missing changes to `k8s.yml`:
https://github.com/okteto/go-getting-started/pull/10/files#diff-9aa5c3faef5c8a780db30dc0dd41b946e5732af5677ee2fc17d3db774fe1a57b
You are also missing to commit the `.stignore` file.

Comment (gabriel-okteto) [reply .gitignore:3]: there was an existing .stignore in this repo, should i rename/delete it and run up to generate a new one or?

**Review by pchico83 — APPROVED** [review]
@gabriel-okteto I think you are missing to commit `.stignore`. Could you check if we committed `.stignore`in our previous samples?

**Pull request merged**